src/mlpack/core/tree/hollow_ball_bound.hpp

```diff
@@ -35,6 +36,8 @@ class HollowBallBound
   math::RangeType<ElemType> radii;
   //! The center of the ball bound.
   arma::Col<ElemType> center;
+  //! The center of the hollow.
+  arma::Col<ElemType> hollowCenter;
```

Comment:
The normal center is the center of the minimal bounding ball (i.e. the outer ball). And `hollowCenter` is the center of the hole (i.e. the inner ball).